### DISCUSSION TITLE
Erro ao pagar QRCode estatico (Corrigido)

### DISCUSSION THREAD

**Discussion by BRomano** [bacen/pix-api#394] [General]
Gerei um QRCode dinamico para identificação e pagamento pelo app do banco, banco identifica pagamento, faz o pix, porem da cancelado, e não finaliza o pix.

Porem quando faço o pix usando a chave direto no app do banco ele funciona.

Ao pesquisar, verifiquei esse site, e ao gerar o QRCode usando ele, esta dando o mesmo erro tbm https://www.gerarpix.com.br/

EMV

00020126580014BR.GOV.BCB.PIX0136E7A3118C-C8A3-443E-A3EB-033304F0DB7252040000530398654040.015802BR5915BDS 3D Planning6008Curitiba62120508TESTEPIX630422E0

![image](https://user-images.githubusercontent.com/4623909/118502375-06638300-b700-11eb-8a21-d2c79a7b7742.png)

QRCode gerado

Para os bancos lerem com sucesso o QRCode, falta alguma informação? É necessário algum template do 80-99 ?

**Comment by renatofrota** [comment]
3 pontos:

1. esse QR Code é estático e não dinâmico;
2. a chave Pix está em letras maiúsculas. Alguns PSPs erroneamente só aceitam a chave em minúsculas (conforme registradas no DICT) como, por exemplo, o Itaú (PSP ao qual a chave está vinculada, inclusive);
3. o maior problema é justamente o PSP recebedor (Itaú). Ele não aceita em nenhuma hipótese QR Codes gerados de forma autônoma (problema relatado [aqui](https://github.com/renatofrota/pix-pendencias/issues/26)).

**Reply by BRomano** [reply]
Obrigado pela resposta.

1. Editei o titulo.
2. Testei com maiusculas e minusculas, deve ser o problema 3 provavelmente.
3. Como gero um QRCode pro ITAU? Tenho que gerar na plataforma deles?
